src/puppetlabs/trapperkeeper/authorization/ring.clj

```diff
@@ -23,3 +15,36 @@
    schema/Keyword schema/Any})
 
 ;; Functions
+
+(schema/defn get-authorized-authentic? :- schema/Bool
```

Comment:
I don't really think the `get-` part of these functions is necessary but not enough to hold up the merge. You want to hit the button, @nwolfe ?